### PR TITLE
Upgrade PackerBuildV1 Task to NodeJS16

### PR DESCRIPTION
**Pull request by ismayilov-ismayil**
**Task name**:  PackerBuildV1 

**Description**: Upgrade PackerBuildV1 Task to NodeJS16

**Documentation changes required:** (Y/N) N

**Added unit tests:** (Y/N) N

**Attached related issue:** (Y/N) N
**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [x] Checked that applied changes work as expected

Success pipeline run
image

### CODE REVIEW

File: Tasks/PackerBuildV1/Strings/resources.resjson/en-US/resources.resjson [hunk 1]
```diff
@@ -89,9 +89,6 @@
   "loc.messages.ParsingAdditionalBuilderParameters": "Parsing additional builder parameters json.",
   "loc.messages.ParsingTemplateFileContentFailed": "Unable to parse json content from template file %s with error: %s.",
   "loc.messages.ParsingCustomTemplateParameters": "Parsing custom template parameters json.",
-  "loc.messages.FetchingSPNDetailsRemotely": "Fetching SPN details for app ID %s from the Azure AD graph service connection...",
```
Comment (vmapetr) [inline Tasks/PackerBuildV1/Strings/resources.resjson/en-US/resources.resjson:92]: Sorry, could you please explain why these strings were removed?

Comment (ismayilov-ismayil) [reply Tasks/PackerBuildV1/Strings/resources.resjson/en-US/resources.resjson:92]: it is not releated my PR, when I run make.js build, it updated this file as well. It seems it is not relevant translation any more

**Review by vmapetr — APPROVED** [review]

**Review by Roman-Shchukin — APPROVED** [review]

**Comment by ismayilov-ismayil** [comment]
@bishal-pdMSFT could you please take a look

**Review by bishal-pdMSFT — APPROVED** [review]

**Pull request closed without merging**